app/common/constants/toolbarUserInterfaceScale.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const scaleSize = {
+  SMALLER: 'smaller',
+  NORMAL: 'normal',
+  LARGER: 'larger',
+  SUPERSIZE: 'supersize'
+}
+
+let zoomLevel = {}
```

Comment:
Object makes sense IMO- I use the array style syntax for accessing the object since the dot notation is not available with a variable.  ex: zoomLevel.SUPERSIZE won't work, hence zoomLevel[SUPERSIZE]